.github/workflows/ci.yaml

```diff
@@ -60,8 +60,8 @@ jobs:
           # recent branch cache.
           # Then it will save a new cache at this commit sha, which should be used by
           # the next build on this branch.
-          key: stack-work-2_${{matrix.os}}-${{github.sha}}
-          restore-keys: stack-work-2_${{matrix.os}}
+          key: stack-work-3_${{matrix.os}}-${{github.sha}}
+          restore-keys: stack-work-3_${{matrix.os}}
```

Comment:
Aside: Any ideas on this?

Reply:
Do you mean any ideas about why the cache gets busted sometimes?

Reply:
Yeah

Reply:
Or more like: is there anything we can do to prevent the cache from getting busted sometimes, without significantly increasing average build time?

Reply:
I'm not sure, maybe cabal wouldn't have these problems 🤷‍♂️